code/modules/cargo/centcom_podlauncher.dm

```diff
@@ -755,6 +750,13 @@
 				else
 					var/atom/movable/movable_to_launch = thing_to_launch
 					movable_to_launch.forceMove(toLaunch) //and forceMove any atom/moveable into the supplypod
+	var/list/nearby_mobs = list()
+	for (var/mob/living/target_mob in range(2, target_turf))
+		nearby_mobs.Add(target_mob)
```

Comment:
Doesn't range already include turf you're on?

Reply:
Yes, nearby includes the targeted turf

Reply:
Yes? It's only used for display purposes as you can see in the screenshots